### DISCUSSION TITLE
client uploads files with extended filename

### DISCUSSION THREAD

**Discussion by Grimmwicht** [abraunegg/onedrive#1465] [Q&A] [answered]
Hi folks,

I'm using this onedrive client on a Onedrive Business, which is used by many people. Sometimes ago some strange files were discovered – duplicates with an added -ze65 at the filename, which is the name of my machine.

I watched this since, but can't find a cause for this behaviour. This seems to happen to random files. First, the client reports a download of a changed file (correct, changed by someone other), later in a second step, it says „Uploading modified file ... skipped” (which is bollocks, because I didn't touch this file), and third, it uploads this file as filename-ze65.extension.

I would love to get some hints, why this happens – mostly the client works fine as expected. Thanks in advance.

**Comment by abraunegg** [comment, marked as answer]
@Grimmwicht 
The file on your local file system gets renamed, because there is a conflict. It is renamed to ensure data integrity, so that you do not lose any data. This mainly occurs when the following events are true:
* The remote file got updated online|by someone else
* You modified the file locally or the client 'thinks' you did based on file timestamps
  * **Note:** There are some *major* bugs with some of the Linux file managers and file indexers, that could be touching your files - modifying them / changing the timestamp, thus the client thinks you have modified them - even if you actually didn't edit the file. See: https://github.com/abraunegg/onedrive/issues/824#issuecomment-603469431 

This is also a very old 'skilion' bug ... so what client version (`onedrive -v`) are you actually using?

Things for you to do:
* Make sure you are running the latest client release or build the client from 'master' at least once a month. Do not rely on distribution packages unless they are highly proactive and kept in line with releases.
* Change the behavior of the client, by updating your configuration file: `bypass_data_preservation = "true"` - as this will prevent the rename action from occurring, but, now puts you at risk of potential data loss if you modify the file locally, and you download an updated file, your local file changes will be lost.

**Reply by Grimmwicht** [reply]
Thanks for your help. I'm using v2.4.11 on Manjaro from the AUR: https://aur.archlinux.org/packages/onedrive-abraunegg

I'll have a look at the links you provided. Thanks so far.

**Reply by abraunegg** [reply]
@Grimmwicht 
Given you are using Manjaro, and the prior issue was also on Manjaro - it is *highly* likely this is the same issue. This is *not* an issue with this client, but with Manjaro and certain components that it runs.
